CHANGES.rst

```diff
@@ -1,5 +1,6 @@
 2.0.0 (Unreleased)
 ==================
+- [FIX] Fixed Cloudant._usage_endpoint to only accept both year and month or neither.
```

Comment:
Since `._usage_endpoint` is private, I think it would be better to frame the change in terms of the API so that when users look at this file they will know how the change affects them.

Something like: `[FIX] Added validation to Cloudant.bill(...), Cloudant.volume_usage(...), and Cloudant.requests_usage(...) to ensure that a valid year/ month combination or neither are used as arguments.`